runtime/pom.xml

```diff
@@ -39,7 +39,7 @@
 
     <!-- Base image to use for Docker images. Pinned to 1.2.3 since 1.2.4 has issues because
          of Java 1.8.0_121 (1.2.3 is running 1.8.0_111) -->
-    <java-base-image>fabric8/java-jboss-openjdk8-jdk:1.2.3</java-base-image>
+    <java-image>fabric8/java-jboss-openjdk8-jdk:1.2.3</java-image>
```

Comment:
You still have to update https://github.com/iocanel/ipaas-rest/blob/341a57ea07d1f4b91e6a0121cc81eb46efadb1d3/runtime/pom.xml#L93 !